goverge/main.py

```diff
@@ -58,18 +58,25 @@ def get_project_package(project_root, project_import):
     return project_import.replace("'", "")
 
 
-def get_test_packages(project_root):
+def get_test_packages(project_root, ignore):
     """
     Get the list of packages in the project
     :type project_root: string
     :param project_root: The location of the project root
     :rtype: List
+    :param ignore: List of directories to ignore
+    :rtype: List
     :return: The list of packages to run coverage on
     """
-    return [
-        x[0] for x in os.walk(project_root)
-        if not any(word in x[0] for word in ["/.", "Godeps", "vendor"])
-        ]
+    ignores = ["/.", "Godeps", "vendor"]
+    if ignore is not None:
+        ignores.extend(ignore)
+    directories = []
+    for root, subdirs, file in os.walk(project_root):
```

Comment:
You have shadowed a builtin python variable with file and I don't think it's really accurate anyways since os.walk() returns a list of files not a single file.